RELEASING.md

```diff
@@ -1,20 +1,18 @@
 # Releasing
 
-Releases of comit-rs are mostly automated.
-Here is what you need to know:
+Releases of comit-rs are mostly automated based on the GitFlow branching model.
 
-The repository uses the GitFlow branching model.
-Hence, releases are started by branching a release-branch (`release/x.y.z`) off `dev`.
+To release a new version, create an issue with the title "Release version x.y.z" and label it with the "release" label.
```

Comment:
❓ What are the implications of making a mistake and using a version number that already exists?

Reply:
The implications are that a PR will be created. If the branch still exists, it will fail to push.

Reply:
Nothing that couldn't be easily recoverable from :)